tern/analyze/docker/dockerfile.py

```diff
@@ -333,3 +335,70 @@ def write_locked_dockerfile(dfile, destination=None):
         file_name = constants.locked_dockerfile
     with open(file_name, 'w') as f:
         f.write(dfile)
+
+
+def check_mutistage_dockerfile(dfobj):
+    """Given a dockerfile object, return if it is a multistage dockerfile."""
+    from_line = [1 for c in dfobj.structure if c['instruction'] == 'FROM']
+    return len(from_line) > 1
```

Comment:
spelling error: `check_multistage_dockerfile`

Straightforward logic is good for readability:
```
342     count = 0                                                                   
343     for c in dfobj.structure:                                                   
344         if c['instruction'] == 'FROM':                                          
345             count = count + 1                                                   
346     return count > 1  
```